scripts/install.sh

```diff
@@ -5,9 +5,17 @@
 IGNORE_ROOT_ARG="--ignore-root-warning"
 IGNORE_ROOT=0
 
-if [ "$1" == "$IGNORE_ROOT_ARG" ]; then
-    IGNORE_ROOT=1
-fi
+while :; do
+    case $1 in
+        $IGNORE_ROOT_ARG) IGNORE_ROOT=1
+        ;;
+        *[a-zA-Z]) ENV=$1
+        ;;
+        *) break
+        ;;
```

Comment:
I'm not sure to understand what's happening in the `case` block, could you explain, please?